src/UnityPackage/Assets/Web3Unity/Scripts/Prefabs/WebGL/WebGLBridge/Web3GL.cs

```diff
@@ -95,7 +93,8 @@ public static void AddTokenFunctionWeb(string tokenAddress,string tokenSymbol,st
             GasLimit = _gasLimit,
             GasPrice = _gasPrice
         };
-        await GameLogger.Log(PlayerPrefs.GetString("ChainId"), PlayerPrefs.GetString("RPC"), data);
+        // Temporarily disabled, waiting for migration to DI
```

Comment:
Maybe it makes sense to add `TODO:` here and on the next occasions too? I don't think VS or Rider supports `FIXME:`, though.